man-roxygen/simple_example.R

```diff
@@ -0,0 +1,20 @@
+<%
```

Comment:
the task_id is not needed in this template. 
Also this simple template should be used where it can be used, sometimes you have exactly the same example written out in the learner docs